src/dodal/beamlines/i04.py

```diff
@@ -410,9 +410,11 @@ def oav_to_redis_forwarder(
     """
     return device_instantiation(
         OAVToRedisForwarder,
-        "oav_to_redis",
+        "oav_to_redis_forwarder",
         "",
```

Comment:
Do we not need a prefix here?

Reply:
> I can't see how the PV prefix is set

`device_instantiation` automatically adds the beamline prefix to the device so if the device part is in the device we can just leave it blank